### PR TITLE
Prepare repository for VAST v2.1.0-rc1

### PR DESCRIPTION
**Pull request by dominiklohmann**
VAST v2.1 is coming soon! This prepares the repository for the first release candidate.

### CODE REVIEW

File: web/blog/2022-06-16-vast-v2.1/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+draft: true
```
Comment (dominiklohmann) [inline web/blog/2022-06-16-vast-v2.1/README.md:2]: Note for the reviewer: This option makes it so the development version of the docs shows this blog post, but it's hidden when building a production version of the docs.

File: web/blog/2022-06-16-vast-v2.1/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+title: VAST v2.1
+description: VAST v2.1 - TODO
```
Comment (dominiklohmann) [inline web/blog/2022-06-16-vast-v2.1/README.md:4]: @mavam this and the other TODOs in this PR are all yours—feel free to just push to this PR directly.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,59 @@ This file is generated automatically. Add individual changelog entries to the 'c
 
 This changelog documents all notable changes to VAST and is updated on every release.
 
+## [v2.1.0-rc1][v2.1.0-rc1]
```
Comment (dominiklohmann) [inline CHANGELOG.md:9]: Optional changelog word-smithing can go in here as well already.

**Comment by dominiklohmann** [comment]
[sc-33611]

File: CHANGELOG.md (hunk 3 above)
Comment (mavam) [reply CHANGELOG.md:9]: Done.

**Review by mavam — APPROVED** [review]
We paired on this.

**Pull request merged**